src/components/Forms/TokenVerifyForm/TokenVerifyForm.js

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { Button } from '@material-ui/core';
+import TextField from '@material-ui/core/TextField';
+import Container from '@material-ui/core/Container';
+import { Redirect } from 'react-router-dom';
+import cookie from 'react-cookies';
+import { makeStyles } from '@material-ui/core/styles';
+import { serviceRequest } from '../../../services/serviceRequest';
+import config from '../../../config';
+
+const useStyles = makeStyles({
+  '@global': {
+    input: {
+      '&::-webkit-outer-spin-button, &::-webkit-inner-spin-button': {
+        '-webkit-appearance': 'none',
+        '-moz-appearance': 'none',
+        margin: 0,
+      },
+      '&[type=number]': {
+        '-webkit-appearance': 'textfield',
+        '-moz-appearance': 'textfield',
+      },
+    },
+  },
+});
+
+const path = '/api/v1/signup/verify';
+const domain = config.apiDomain;
+
+function preparePayload(method, data) {
+  const url = `https://${domain}${path}`;
+  return {
+    method,
+    url,
+    data,
+  };
+}
+
+export const TokenVerifyForm = () => {
+  const classes = useStyles();
+  const [tokenData, setTokenData] = useState({ username: cookie.load('username'), token: '' });
+  const [redirectValid, setRedirectValid] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  function handleChange(event) {
+    const updateTokenData = { ...tokenData };
+    updateTokenData[event.target.name] = event.target.value;
+    setTokenData(updateTokenData);
+  }
```

Comment:
You are overkill this function, because you only have one input, then you even dont need to check what the target.name should be

Reply:
Even thought this won't change the value of target.username, but without calling it in the username inputField, the unit test will fail because no value will be mock into the username inputField, then username return undefined/null

Reply:
I mean rewirte the function to
```  
function handleToekenChange(event) {
    const updateTokenData = { ...tokenData };
    updateTokenData[token] = event.target.value;
    setTokenData(updateTokenData);
  }
```